workflows/post_process_run/fv3post/post_process.py

```diff
@@ -164,6 +170,12 @@ def post_process(rundir: str, destination: str, chunks: str):
     else:
         chunks = {}
 
+    if skip:
+        with open(skip) as f:
```

Comment:
If you use `find`, then the output will be one file per line of f
```suggestion
        with open(skip) as f:
            skip = [line.strip() for line in f.readlines()]
```